modules/abstract-utxo/src/abstractUtxoCoin.ts

```diff
@@ -415,10 +415,15 @@ export abstract class AbstractUtxoCoin extends BaseCoin {
       throw new Error('deprecated');
     }
 
-    const formats = param && param.anyFormat ? undefined : ['default' as const];
+    const formats = utxolib.addressFormat.addressFormats.filter((format) =>
+      utxolib.addressFormat.isSupportedAddressFormat(format, this.network)
+    );
     try {
       const script = utxolib.addressFormat.toOutputScriptTryFormats(address, this.network, formats);
-      return address === utxolib.address.fromOutputScript(script, this.network);
```

Comment:
you can just remove this line - `toOutputScriptsTryFormats` will fail if it cannot parse the address with any of the formats